mailbox/api/src/main/java/org/apache/james/mailbox/acl/UnionMailboxACLResolver.java

```diff
@@ -266,28 +270,24 @@ public MailboxACL applyGlobalACL(MailboxACL resourceACL, boolean resourceOwnerIs
      * authenticated.
      */
     @Override
-    public Rfc4314Rights[] listRights(EntryKey queryKey, GroupMembershipResolver groupMembershipResolver, String resourceOwner, boolean resourceOwnerIsGroup) throws UnsupportedRightException {
+    public List<Rfc4314Rights> listRights(EntryKey queryKey, GroupMembershipResolver groupMembershipResolver, String resourceOwner, boolean resourceOwnerIsGroup) throws UnsupportedRightException {
         Rfc4314Rights[] positiveNegativePair = { MailboxACL.NO_RIGHTS, MailboxACL.NO_RIGHTS };
```

Comment:
it is `{ MailboxACL.NO_RIGHTS, MailboxACL.NO_RIGHTS }` duplicated, why?

Reply:
because it's supposed to be a pair maybe but I don't really know, it's not part of the refactoring